pkg/model/lattice/servicenetwork.go

```diff
@@ -21,11 +21,12 @@ type ServiceNetwork struct {
 
 type ServiceNetworkSpec struct {
 	// The name of the ServiceNetwork
-	Name           string `json:"name"`
-	Namespace      string `json:"namespace"`
-	Account        string `json:"account"`
-	AssociateToVPC bool
-	IsDeleted      bool
+	Name             string    `json:"name"`
+	Namespace        string    `json:"namespace"`
+	Account          string    `json:"account"`
+	SecurityGroupIds []*string `json:"securityGroupIds"`
```

Comment:
aws vpc lattice go sdk use the `[]*string` type to pass the security group ids, keep it in the same type here